docs/api-reference/reference.asciidoc

```diff
@@ -1598,6 +1628,7 @@ SystemRailsConsoleSpec configures the App component of System
 [cols="25a,75a", options="header"]
 |===
 | Field | Description
+| *`image`* __xref:{anchor_prefix}-github-com-3scale-saas-operator-api-v1alpha1-imagespec[$$ImageSpec$$]__ | Image specification for the Sphinx component. Defaults to system image if not defined.
```

Comment:
```suggestion
| *`image`* __xref:{anchor_prefix}-github-com-3scale-saas-operator-api-v1alpha1-imagespec[$$ImageSpec$$]__ | Image specification for the Console component. Defaults to system image if not defined.
```

Reply:
Fixed in 91e729a0f4cd3e1b4006170bd2fa302dbf1c3768

Reply:
Fixed in https://github.com/3scale-ops/saas-operator/commit/88e06aa1ef222ee65548ee97056e514b6a22660b